cli/command/formatter/licenses.go

```diff
@@ -0,0 +1,103 @@
+package formatter
+
+import (
+	"time"
+
+	"github.com/docker/cli/cli/licenseutils"
+)
+
+// TODO - derived from plugin.go - replicate plugin_tests.go before PR
```

Comment:
Is this TODO comment still needed?

Reply:
There's no unit coverage for this file yet, but it's likely going to be tweaked as the upstream license lib is being refined concurrently, so I was delaying that until the new licensing code lands (hopefully tomorrow)